security/fma-shared-lockbox.md

```diff
@@ -0,0 +1,147 @@
+# Shared Lockbox: Failure Modes and Recovery Path Analysis
+
+| Author | Joxes, Agusduha |
+| --- | --- |
+| Created at | 2025-01-28 |
+| Initial Reviewers | Pending |
+| Needs Approval From | Pending |
```

Comment:
@smartcontracts adding you as a second "needs approval from" reviewer here since the lockbox is a huge honeypot and very critical
```suggestion
| Initial Reviewers | Josep Bove |
| Needs Approval From | Matt Solomon, Kelvin Fichter |
```